pocs/utils/rs232.py

```diff
@@ -39,14 +39,45 @@ def get_serial_port_info():
     return sorted(get_comports(), key=operator.attrgetter('device'))
 
 
-class SerialData(PanBase):
+class SerialData(object):
     """SerialData wraps a PySerial instance for reading from and writing to a serial device.
 
     Because POCS is intended to be very long running, and hardware may be turned off when unused
     or to force a reset, this wrapper may or may not have an open connection to the underlying
     serial device. Note that for most devices, is_connected will return true if the device is
     turned off/unplugged after a connection is opened; the code will only discover there is a
     problem when we attempt to interact with the device.
+
+    .. doctest::
```

Comment:
Yippee!

Reply:
These *should* also be counting toward our coverage but that's yet to be verified.